src/modules/map/components/infoButton/InfoButton.js

```diff
@@ -29,18 +29,25 @@ export class InfoButton extends BaElement {
 			{ name: translate('map_info_button_about'), url: 'https://geoportal.bayern.de/geoportalbayern/seiten/impressum.html' }
 		];
 
-		const openPopup = () => {
-
+		const openPopup = () => {			
 			// set created popup visible
 			var popup = this.shadowRoot.getElementById('info-popup');
-			if (popup.getAttribute('type') === 'hide') {
-				popup.setAttribute('type', 'show');
+			if (!popup.isOpen()) {
+				popup.openPopup();
 			}
 			else{
-				popup.setAttribute('type', 'hide');
+				popup.closePopup();
 			} 
```

Comment:
Is this clause really needed? If true, we also need a test for this clause and we should rename the function ti something
like `togglePopup`

Reply:
That's my fault. This isn't needed. I'll remove that.